### PR TITLE
Update PAO manifests to use 4.10 images

### PR DESCRIPTION
**Pull request by cynepco3hahue**

### CODE REVIEW

File: feature-configs/deploy/performance/operator_catalogsource.yaml [hunk 1]
```diff
@@ -10,6 +10,6 @@ spec:
   icon:
     base64data: ""
     mediatype: ""
-  image: quay.io/openshift-kni/performance-addon-operator-index:4.9-snapshot
+  image: quay.io/openshift-kni/performance-addon-operator-index:4.10-snapshot
```
Comment (bartwensley) [inline feature-configs/deploy/performance/operator_catalogsource.yaml:13]: Why is a separate catalog required for the PAO? It seems the PAO is present in the redhat-operators catalog and the OCP documentation [1] points there as well.

[1] https://access.redhat.com/documentation/en-us/openshift_container_platform/4.9/html/scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes#install-operator-cli_cnf-master)

Comment (cynepco3hahue) [reply feature-configs/deploy/performance/operator_catalogsource.yaml:13]: The cnf-features-deploy should point to the 4.10 PAO image that is still not released and does not exist under the redhat-operators.

Comment (bartwensley) [reply feature-configs/deploy/performance/operator_catalogsource.yaml:13]: OK - thanks for the explanation.

**Review by SchSeba — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**